web/src/cypress/integration/services.ts

```diff
@@ -547,20 +547,38 @@ function testServices(screen: ScreenFormat) {
       }),
     )
 
-    it('should allow creating alerts', () => {
+    it('should create alert with prepopulated service', () => {
       cy.pageFab()
+      cy.get('div[role=dialog]').as('dialog')
+
       const summary = c.sentence({ words: 3 })
       const details = c.word({ length: 10 })
       cy.get('input[name=summary]').type(summary)
       cy.get('textarea[name=details]').type(details)
 
-      cy.get('*[role=dialog]')
-        .contains('button', 'Submit')
+      cy.get('@dialog')
+        .contains('button', 'Next')
         .click()
 
-      cy.location('pathname').should('contain', '/alerts/') // details page
+      // service already selected
+      cy.get('@dialog')
```

Comment:
If we're using the new flow/dialog, can we just skip/omit the second step entirely and have it show on the confirm step?

Reply:
Yeah, we can just go from info -> confirm I think -- it would be confusing if there was a step you couldn't get to or was read-only